src/index.html

```diff
@@ -2,16 +2,160 @@
+          Name:
+          <input type="text" name="name">
+        </label>
+
+        <label class="form_field">
+          How old are You?
+          <input 
+            type="number" 
+            name="old" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+
+        <label class="form_field">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date"
+          >
+        </label>
+        <div class="form_field">
+          <label for="terms-and-conditions" >
+            I accept the term of the agreement
+          </label>
+          <input 
+            type="checkbox" 
+            name="trems" 
+            id="terms-and-conditions" 
+            class="field"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+          <input 
+            type="email" 
+            placeholder="email@example.com" 
+            name="email" 
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="6" 
+            maxlength="20" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="no"
+            >
+            No
+          </label>
+        </label>
+
+        <label class="form_field">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="color"
+          >
+        </label>
+
+        <label class="form_field">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="time" 
+            step="2"
+          >
+        </label>
+
+        <label class="form_field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form_field">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="range" 
+            min="1" 
+            max="5" 
+            value="-1"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label class="form_field">
```

Comment:
Don't forget about empty lines between  multilane children
```suggestion
        <legend>Additional info:</legend>
        
        <label class="form_field">
```

Reply:
ok